client/src/app/shared/models/agenda/item.ts

```diff
@@ -61,6 +61,17 @@ export class Item extends ProjectableBaseModel {
         return this.speakers.filter(speaker => speaker.state === SpeakerState.WAITING).length;
     }
 
+    /**
+     * Gets the string representation of the item type
+     */
```

Comment:
Missing `@returns` statement

(Even documentation is not required in this class, it is nice to have some for functions like this one)